### ISSUE TITLE
Is this repo out-of-date?

### ISSUE THREAD

**Issue by kerryjiang** [Azure/azure-cosmos-db-emulator-docker#72]
I noticed the image here mcr.microsoft.com/cosmosdb/windows/azure-cosmos-emulator seems not consistent with the code in this repo.

For example, the startup script is Start.ps1 instead of startemu.cmd in this repo.

Directory of C:\CosmosDB.Emulator 

03/07/2022  06:55 PM    <DIR>          .
03/07/2022  06:55 PM    <DIR>          ..
06/18/2023  12:25 PM    <SYMLINKD>     bind-mount [\\?\ContainerMappedDirectories\7BEBBCB6-E5FE-434D-B6CC-CA0BDA5A5791]
03/07/2022  06:34 PM             7,209 Start.ps1
               1 File(s)          7,209 bytes
               3 Dir(s)  131,457,888,256 bytes free

**Comment by diegosasw** [comment]
One year later. Any news on this? Is this repo being maintained? I'm unable to work with docker emulator and I still don't know why. The Microsoft documentation is certainly not accurate with the reality of the cosmos db emulator.

**Comment by sajeetharan** [comment]
Hi @diegosasw Thanks for reaching out, we are working on addressing the concerns. Yes this repo is still active and maintained, can you please specify your issue?

**Comment by diegosasw** [comment]
Hi @sajeetharan, thanks for the clarification.

It's this https://github.com/Azure/azure-cosmos-db-emulator-docker/issues/101

Basically I tried to follow Microsoft current documentation on CosmosDb emulator using the docker version and a simple console application but without any luck in creating a simple database. Not sure if documentation is out of date or if there is any current gotcha with the docker emulator.

I decided to reply on this thread because I also was wondering what the current status with this repo is.
